app/page.tsx

```diff
@@ -1,25 +1,63 @@
-"use client"
-
-import React from "react";
-import AsideMenu from "./Components/Aside/Aside";
+"use client";
+import React, { Component, useState } from "react";
 import styles from "./page.module.scss";
-import Navigation from "./Components/Navigation/Nav";
+import PlayerController from "./Components/PlayerController/PlayerController";
 import MusicListItem from "./Components/MusicList/MusicListItem";
+import songs from "@/public/Consts/songs"; 
+import { RecoilRoot } from "recoil";
 
 const Home = () => {
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [currentTime, setCurrentTime] = useState(0);
+  const [showPlayer, setShowPlayer] = useState(true);
```

Comment:
and it should be recoil state as Player should be in layout i think

Reply:
i didnt understand

Reply:
player should be in layout